src/TableSearchHeader.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const TableSearchHeader = ({kingdomOptions, entriesPerPageOptions, totalNumberOfRows, searchAllOnChange, numberOfEntriesPerPageOnChange, kingdomOnChange}) =>
+const TableSearchHeader = ({dropdownFilters, entriesPerPageOptions, totalNumberOfRows, searchAllOnChange, numberOfEntriesPerPageOnChange, dropdownFiltersOnChange}) =>
   <div className={`row expanded`}>
-    <div className={`small-12 medium-4 large-2 columns`}>
-      <label> Kingdom:
-        <select defaultValue={``} onChange={e => kingdomOnChange(e)}>
-          <option value={``} >All</option>
-          {
-            kingdomOptions.map(kingdom =>
-              <option key={kingdom} value={kingdom}>
-                {kingdom.charAt(0).toUpperCase() + kingdom.slice(1)}
-              </option>) 
-          }
-        </select>
-      </label>
-    </div>
-
+    {
+      dropdownFilters.map((dropdownFilter) => {
+        return (<div key={dropdownFilter.label} className={`small-12 medium-4 large-2 columns`}>
```

Comment:
This `map` function could be reformated as `dropdownFilter => <div>...</div>`.

Reply:
Modified it.